test/reline/helper.rb

```diff
@@ -22,29 +22,24 @@ module Reline
   class <<self
     def test_mode(ansi: false)
       @original_iogate = IOGate
-      remove_const('IOGate')
-      const_set('IOGate', ansi ? Reline::ANSI : Reline::GeneralIO)
+      @original_screen_size = @original_iogate.get_screen_size
```

Comment:
I think this `@original_screen_size` and `IOGate.set_screen_size(*@original_screen_size)` is not needed because we don't modify `@original_iogate` anymore.

(perhaps this is causing windows test failure)